extensions/interactions/baseValidator.js

```diff
@@ -64,8 +64,8 @@ oppia.factory('baseInteractionValidationService', [
           partialWarningsList.push({
             type: WARNING_TYPES.ERROR,
             message: (
-              'Please add feedback for the user if they are to return to the ' +
-              'same state again.')
+              'Please add feedback for the user in the [All other answers] ' +
+              'rule if they are to return to the same state again.')
```

Comment:
I suggest removing "if they are to return to the same state again" as currently it sounds like a conditional - i.e. the editor has to determine whether the user would return to the same state again, and if so fix things.